### PR TITLE
VMs: bump to f41

### PR DESCRIPTION
**Pull request by edsantiago**
Cleaned-up version of #390 because that was a mess of iterative try-this-and-that

### CODE REVIEW

**Comment by edsantiago** [comment]
|  |*debian*|*prior-fedora*|*fedora*|*fedora-aws*|*rawhide*|
|----|----|----|----|----|----|
|*base*         | 13.5            | Generic         | Generic-41-1.4  | ?               | 42-0            |
|*kernel*       | 6.11.6-1        | 6.8.5-301       | 6.11.6-300      | 6.11.5-300      | 6.12.0-0.rc6.51 |
|*grub2-common* | 2.12-5          | 2.06-123        | 2.12-10         | 2.12-10         | 2.12-10         |
|*aardvark-dns* | 1.12.2-1        | 1.12.2-2        | 1.13.0-1        | 1.13.0-1        | 1.12.2-2        |
|*netavark*     | 1.12.1-3        | 1.12.2-1        | 1.13.0-1        | 1.13.0-1        | 1.12.2-1        |
|*buildah*      | 1.37.5+ds1-1    | 1.37.3-1        | 1.37.5-1        | 1.37.5-1        | 1.37.5-1        |
|*conmon*       | 2.1.12-3        | 2.1.12-2        | 2.1.12-3        | 2.1.12-3        | 2.1.12-3        |
|*container-selinux*| ?               | 2.233.0-1       | 2.233.0-1       | 2.233.0-1       | 2.233.0-1       |
|*containers-common*| ?               | 0.60.4-2        | 0.60.4-4        | 0.60.4-4        | 0.60.4-5        |
|*criu*         | 3.19-2          | 4.0-1           | 4.0-2           | 4.0-2           | 4.0-2           |
|*crun*         | 1.18.2-1        | 1.17-1          | 1.18.2-1        | 1.18.1-1        | 1.18.2-1        |
|*docker-ce*    | 5:27.3.1-1\~debian.12\~bookworm | ?               | ?               | ?               | ?               |
|*golang*       | 2:1.23\~2       | 1.22.7-1        | 1.23.2-2        | 1.23.2-2        | 1.23.2-2        |
|*gvisor-tap-vsock*| ?               | ?               | ?               | ?               | ?               |
|*nmap-ncat*    | 7.94+git20230807.3be01efb1+dfsg-4 | 7.92-2          | 7.92-3          | 7.92-3          | 7.92-3          |
|*passt*        | 2024-10-30      | 2024-09-06      | 2024-10-30      | 2024-10-30      | 2024-10-30      |
|*podman*       | 5.2.5-1         | 5.2.3-1         | 5.3.0\~rc2-1    | 5.2.5-1         | 5.3.0\~rc2-1    |
|*runc*         | 1.1.15+ds1-1    | 1.1.12-3        | 1.2.1-1         | 1.1.12-4        | 1.2.1-1         |
|*skopeo*       | 1.16.1+ds1-1+b1 | 1.16.1-1        | 1.16.1-1        | 1.16.1-1        | 1.16.1-2        |
|*slirp4netns*  | 1.2.1-1+b1      | 1.2.2-2         | 1.3.1-1         | 1.2.2-3         | 1.3.1-1         |
|*systemd*      | 256.7-3         | 255.13-1        | 256.7-1         | 256.7-1         | 256.7-1         |
|*tar*          | 1.34+dfsg-1.2+deb12u1 | 1.35-3          | 1.35-4          | 1.35-4          | 1.35-4          |

**Review by Luap99 — COMMENTED** [review]
changes LGTM

File: cache_images/fedora_packaging.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ req_env_vars PACKER_BUILD_NAME
 if [[ "$PACKER_BUILD_NAME" == "fedora" ]] && [[ ! "$PACKER_BUILD_NAME" =~ "prior" ]]; then
     warn "Enabling updates-testing repository for $PACKER_BUILD_NAME"
     lilto ooe.sh $SUDO dnf install -y 'dnf-command(config-manager)'
-    lilto ooe.sh $SUDO dnf config-manager --set-enabled updates-testing
+    lilto ooe.sh $SUDO dnf config-manager setopt updates-testing.enabled=1
```
[the remark this answers is not in the document]
Comment (edsantiago) [reply cache_images/fedora_packaging.sh:31]: What do you recommend?

[the remark this answers is not in the document]
Comment (Luap99) [reply cache_images/fedora_packaging.sh:31]: I don't know the history behind why we have such difference there, I have not chased the git history to end but it is there for many years so maybe better not touch that? I mean we have a workaround in podman for the kernel problem

**Comment by edsantiago** [comment]
Now [passing in buildah](https://github.com/containers/buildah/pull/5820) thanks to Paul.

I'd like to get this merged, so we can go through this again next week and see if we pick up new kernel and new pasta.

**Comment by Luap99** [comment]
yes agree

**Pull request merged**